wasmer/instance.go

```diff
@@ -1,5 +1,6 @@
 package wasmer
 
+// #include <stdlib.h>
```

Comment:
please remove comment

Reply:
this is not a comment. this is a c include - it is a must for C.free

Reply:
Ok